### ISSUE TITLE
Installation script is broken for SUSE distributions

### ISSUE THREAD

**Issue by bartmichu** [zerotier/ZeroTierOne#1379]
When detecting Linux distribution, installation script assumes ```/etc/SuSE-release``` file exists for SUSE distros. That fails on new versions because it has been removed.

```
toor@HOLODECK:~> lsb_release -a
LSB Version:	n/a
Distributor ID:	openSUSE
Description:	openSUSE Tumbleweed
Release:	20210325
Codename:	n/a
toor@HOLODECK:~> curl -s https://install.zerotier.com | sudo bash
[sudo] password for root: 

*** ZeroTier One Quick Install for Unix-like Systems

*** Tested distributions and architectures:
***   MacOS (10.7+) on x86_64 (just installs ZeroTier One.pkg)
***   Debian (7+) on x86_64, x86, arm, and arm64
***   RedHat/CentOS (6+) on x86_64 and x86
***   Fedora (16+) on x86_64 and x86
***   SuSE (12+) on x86_64 and x86
***   Mint (18+) on x86_64, x86, arm, and arm64

*** Please report problems to contact@zerotier.com and we will try to fix.

*** Detecting Linux Distribution


*** Package installation failed! Unfortunately there may not be a package
*** for your architecture or distribution. For the source go to:
*** https://github.com/zerotier/ZeroTierOne
```

## Additional information 

OpenSUSE documentation has an [article for third-party software developers](https://en.opensuse.org/SDB:SUSE_and_openSUSE_Products_Version_Outputs), who wish to add SUSE and openSUSE distros compatibility into their software.

In short, ```/usr/lib/os-release``` is the way to go.
